### PR TITLE
[NETBEANS-1190] Add more icon sizes to NetBeans' MainWindow

### PR DESCRIPTION
**Pull request by ralphbenjamin**
Extraced icons from MacOSX Launcher's icns and included them for the MainWindow/Splash

### CODE REVIEW

**Comment by geertjanw** [comment]
Any reason not to merge this, looks great to me, I will wait 24 hours for objections, if there are none, I will merge this.

File: platform/core.windows/src/org/netbeans/core/windows/view/ui/MainWindow.java [hunk 1]
```diff
@@ -504,14 +504,20 @@ private static Border getDesktopBorder () {
...
        if( !currentIcons.isEmpty() )
            return; //do not override icons if they have been already provided elsewhere (JDev)
        f.setIconImages(Arrays.asList(
                ImageUtilities.loadImage(ICON_16, true),
                ImageUtilities.loadImage(ICON_32, true),
-               ImageUtilities.loadImage(ICON_48, true)));
+               ImageUtilities.loadImage(ICON_48, true),
+                ImageUtilities.loadImage(ICON_256, true),
+                ImageUtilities.loadImage(ICON_512, true),
+                ImageUtilities.loadImage(ICON_1024, true)));
```
Comment (junichi11) [inline platform/core.windows/src/org/netbeans/core/windows/view/ui/MainWindow.java]: Wrong indentation spaces?

**Comment by junichi11** [comment]
Please squash and merge when we merge this PR. Thanks!

**Comment by matthiasblaesing** [comment]
Thank you  - merging.

**Pull request merged**